wasi/2024/WASI-04-04.md

```diff
@@ -29,4 +29,5 @@ The meeting will be on a zoom.us video conference.
     1. _Submit a PR to add your announcement here_
 1. Proposals and discussions
     1. Vote: Advance `wasi-i2c` proposal to Phase 2
-    1. _Submit a PR to add your announcement here_
+    3. _Submit a PR to add your announcement here_
```

Comment:
Probably didn't mean to edit this line. But this ok, I'm in the process of adding the notes now